playwright/helpers/createUniqueDocument.ts

```diff
@@ -0,0 +1,18 @@
+import {SanityDocument, SanityDocumentStub} from '@sanity/client'
+import {uuid} from '@sanity/uuid'
+import {testSanityClient} from '.'
+
+export async function createUniqueDocument({
+  _type,
+  _id,
+  ...restProps
+}: SanityDocumentStub): Promise<Partial<SanityDocument>> {
+  const doc = {
+    _type,
+    _id: _id || uuid(),
+    ...restProps,
```

Comment:
Could we add a unique ID for this test run here, for further isolation? We should probably add it to the CI schemas as a hidden string field as well, if we do.

```suggestion
    ...restProps,
    runId: TEST_RUN_ID,
```